src/etc/inc/auth.inc

```diff
@@ -593,8 +610,18 @@ function local_user_del($user) {
 
 function local_user_set_password(&$user, $password) {
 
-	$user['password'] = crypt($password);
-	$user['md5-hash'] = md5($password);
+	$user['password'] = "REMOVED";
+	$user['md5-hash'] = "REMOVED";
```

Comment:
I left the fields in just in case something was relying upon them for something, but if that's not the case I'll unset them.